cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -44,4 +44,12 @@ spec:
           - name: POD_NAMESPACE
             valueFrom:
               fieldRef:
-                fieldPath: metadata.namespace
\ No newline at end of file
+                fieldPath: metadata.namespace
+        volumeMounts:
+          - mountPath: /cache
+            name: package-cache
+      volumes:
+      - name: package-cache
+        emptyDir:
+          medium: {{ .Values.packageCache.medium }}
+          sizeLimit: {{ .Values.packageCache.sizeLimit }}
```

Comment:
We probably want to be able to mount an existing volume here (that may already have some packages cached). That being said, I think it could also be a follow-up PR.